wc-i18n.html

```diff
@@ -75,7 +75,24 @@
      * 
      * `WCI18n` is included and used in your component as follows:
      * 
-     * ```html
+     *```Polymer 2 html
+     * <dom-module id='custom-el'>
+     *    <template>
+     *      <!-- Use the provided `i18n` function -->
+     *      <p>i18n('key')</p>
```

Comment:
I wish I'd updated in my PR as well... this line needs braces `[[ ]]` around the i18n function.